srcpkgs/lsp-plugins/template

```diff
@@ -21,6 +21,55 @@ do_configure() {
 	make LD="$CXX" PREFIX=/usr config
 }
 
+lsp-plugins-lib_package() {
```

Comment:
standard package naming would be `-libs`

Reply:
Done.